lib/src/document.dart

```diff
@@ -65,8 +67,11 @@ class Document {
     }
   }
 
+  /// Parses the given [text] to a series of AST nodes.
+  List<Node> parse(String text) => parseLines(text.toLines());
+
   /// Parses the given [lines] of Markdown to a series of AST nodes.
-  List<Node> parseLines(List<String> lines) {
+  List<Node> parseLines(List<Line> lines) {
```

Comment:
Might want to add a NEW function here called `parseLineList` or similar. This maintains backwards compat.

Thoughts @srawlins ?

Reply:
I have added  `parse(String text)`, which accepts  a string parameter.

Reply:
We need to MAINTAIN the existing API form.

Or figure out a work-around for Flutter.

Maybe we release v7 and deprecate the current API name and add a new name. Etc etc...releasing software is complicated!

Reply:
Now I got it!